tioga-build-and-test.yml

```diff
@@ -73,9 +73,9 @@ status_pending:
     - export pipeline_status="pending"
     - *status_report
 
-rocm_5_1_1_clang_13_0_0:
+rocm_5_3_0_clang_13_0_0:
```

Comment:
👍

Reply:
@kab163 you may have to create a PR in radiuss-spack-configs too to reference the appropriate rocm install.

Reply:
So given that I merged those changes to the radiuss-spack-configs, is there something needed here (in this repo) to pull in those changes? Or will it just be ok?

Reply:
You only need to update the radiuss-spack-configs in the project repo (here RAJA).
 There is an implicit need for coherency between radiuss-spack-confg and radiuss shared CI.